products/extensions/b2b-components/employee-management/concept/README.md

```diff
@@ -0,0 +1,16 @@
+# Concept
+
+## Basic idea
+
+Employee management, as part of the B2B components, is a feature that allows you to manage employees and their permissions as an extension to Shopware's account and customer management, but gathered in a company context. This means that employees are associated with a **company customer** and will act on behalf of that company, when talking about e.g. placing orders in the shop using addresses and payment methods defined by administrators of such company.
```

Comment:
```suggestion
Employee management, as part of the B2B components, is a feature that allows you to manage employees and their permissions as an extension to Shopware's account and customer management, but gathered in a company context. This means that employees are associated with a **company customer** and will act on behalf of that company when, for example, placing orders using addresses defined by administrators of such company.
```

Let's keep the payment methods out of this for now, otherwise it might be misleading regarding the current feature set.